README.md

```diff
@@ -1,3 +1,4 @@
+x 
 <p align="center">
 <img src="doc/images/sirius_logo.png" width="500">
```

Comment:
Bench should run on repo = haampie/SIRIUS, sha = 3a5dfbdc503ee17017918d5d6ddb109e96364a94, url = https://github.com/haampie/SIRIUS/pull/1#discussion_r499729158, fromkind = review, comment_kind = review